app/workers/workflow_retired_count_worker.rb

```diff
@@ -11,7 +11,7 @@ class WorkflowRetiredCountWorker
     key: ->(workflow_id) { "workflow_#{workflow_id}_retired_count_worker" }
   }
 
-  sidekiq_options unique: :until_executing
+  sidekiq_options unique: :until_executed
```

Comment:
this could also be handled by modifying the congestion interval but we may not know the length of the job